be/src/storage/lake/vacuum.cpp

```diff
@@ -70,19 +53,24 @@ static Status delete_files(FileSystem* fs, const std::vector<std::string>& paths
     if (wait_duration > 0) {
         std::this_thread::sleep_for(std::chrono::milliseconds(wait_duration));
     }
-    for (auto&& path : paths) {
-        LOG_IF(INFO, config::lake_print_delete_log) << "Deleting " << path;
+
+    if (config::lake_print_delete_log) {
+        for (size_t i = 0, n = paths.size(); i < n; i++) {
+            LOG(INFO) << "Deleting " << paths[i] << "(" << (i + 1) << '/' << n << ')';
```

Comment:
this line is very chatty, do we have ways to compress the log output, e.g. only record the filename, reduce the filepath common prefixes.

Reply:
I haven't come up with a good idea yet